src/gui/windows/multiplayer.zig

```diff
@@ -56,14 +56,17 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
+			// TODO: Show error message in the GUI
+			std.log.err("IP address cannot be empty", .{});
+			return;
+		}
 		_connection.world = &main.game.testWorld;
 		main.globalAllocator.free(settings.lastUsedIPAddress);
 		settings.lastUsedIPAddress = main.globalAllocator.dupe(u8, ipAddressEntry.currentString.items);
 		settings.save();
 		main.game.world = &main.game.testWorld;
-		main.game.testWorld.init(ipAddressEntry.currentString.items, _connection) catch |err| {
```

Comment:
Without this it won't open the world correctly, and will likely crash further down the line when attempting to actually join a game.